src/index.html

```diff
@@ -7,24 +7,35 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="assets/Roboto-Regular.ttf">
-  <link rel="stylesheet" href="assets/jsonforms-example.css">
-  <link rel="stylesheet" href="assets/bootstrap.css">
   <link rel="stylesheet" href="example.css">
   <script src="assets/native-shim.js"></script>
+
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.1/css/materialize.min.css">
```

Comment:
like this, you need internet to run a local demo. Use local files if possible here and in the following lines